jetbrains-core/tst/software/aws/toolkits/jetbrains/services/schemas/code/DownloadCodeForSchemaDialogTest.kt

```diff
@@ -163,8 +170,17 @@ class DownloadCodeForSchemaDialogTest {
         verify(schemaCodeDownloader).downloadCode(eq(request), any())
         assertThat(fileEditorManager.openFiles).hasOnlyOneElementSatisfying { assertThat(it.name).isEqualTo(fileName) }
 
-        assertThat(errorNotification?.content).isNull()
-        assertThat(infoNotification?.content).isEqualTo(message("schemas.schema.download_code_bindings.notification.finished", SCHEMA_NAME))
+        assertThat(notificationListener.notifications)
+            .filteredOn { it.type == NotificationType.ERROR }
+            .isEmpty()
+
+        assertThat(notificationListener.notifications)
+            .filteredOn { it.type == NotificationType.INFORMATION }
+            .extracting(Function { t -> t.content })
```

Comment:
is the type needed here to resolve ambiguity?

Reply:
Ya, and doing `Noficiation::getContent` also uses wrong method 0.o